### PR TITLE
Refresh plugin

### PR DESCRIPTION
**Pull request by timja**
cc @fr123k

extracted mostly from your PR, as I need to make a change to this plugin and didn't want to do it on top of your massive PR

### CODE REVIEW

**Review by oleg-nenashev — APPROVED** [review]
All changes look good. Thanks for avoiding the binary compatibility changes for now. This codebase is really dated, and a larger push would be required to modernize it.

File: src/main/java/jenkins/advancedqueue/sorter/SorterStrategyDescriptor.java
Comment (timja) [inline src/main/java/jenkins/advancedqueue/sorter/SorterStrategyDescriptor.java:31]: this was breaking for invalid 'html', when I went to check the correct syntax the top 2 articles just recommended removing them, e.g. https://www.vojtechruzicka.com/stop-using-javadoc-author-tag/

any objection @oleg-nenashev

**Comment by timja** [comment]
build is green now

**Review by fr123k — APPROVED** [review]
I did a local test and everything green. LGTM

I will rebase my #50 after the merge.

@timja I'am quite new to jenkins-ci repositories.
Can i approach you for questions and help.

Thanks

**Comment by timja** [comment]
> @timja I'am quite new to jenkins-ci repositories.
> Can i approach you for questions and help.

In general the preference is to use mailing lists and chat rooms rather than individual help, this helps prevent people getting overwhelmed and being blocked on a single person, I would prefer any questions are asked in the open and feel free to point me towards them if you need help and I can try but I do have limited time.

**Comment by oleg-nenashev** [comment]
OK, time to merge it then

**Pull request merged**
